config-templates/aws/aws-ebs-csi-driver/0.8.2/deployment.yaml

```diff
@@ -545,9 +545,7 @@ items:
               args:
                 - --csi-address=$(ADDRESS)
                 - --v=5
-                #{{#enable-volume-scheduling}}
                 - --feature-gates=Topology=true
-                #{{/enable-volume-scheduling}}
```

Comment:
Have you tested this . The correct syntax is 

```
    {{#extra-create-metadata}}
             - --extra-create-metadata
   {{/extra-create-metadata}}
   
   ```

Reply:
These  are commented lines to  enable  volume scheduling always by default.
Removing those commented line (starting with #) wont do anything

Reply:
With fix
```
  csi-provisioner:
    Container ID:  cri-o://cc6fd7c59cb7636c53f8b2ca9507bbfe5cabe89d7356ce5ee61612af5f83fcaa
    Image:         quay.io/k8scsi/csi-provisioner:v1.6.0
    Image ID:      quay.io/k8scsi/csi-provisioner@sha256:78e3393f5fd5ff6c1e5dada2478cfa456fb7164929e573cf9a87bf6532730679
    Port:          <none>
    Host Port:     <none>
    Args:
      --csi-address=$(ADDRESS)
      --v=5
      --feature-gates=Topology=true
      --enable-leader-election
      --leader-election-type=leases
    State:          Running
      Started:      Wed, 24 Feb 2021 16:33:51 +0530
 ```
 
 And without fix
 ```
   csi-provisioner:
    Container ID:  cri-o://c21cf0937364622be8ad4d508494b5c0e3f6666c16fdc1fe77124828721abb28
    Image:         quay.io/k8scsi/csi-provisioner:v1.6.0
    Image ID:      quay.io/k8scsi/csi-provisioner@sha256:78e3393f5fd5ff6c1e5dada2478cfa456fb7164929e573cf9a87bf6532730679
    Port:          <none>
    Host Port:     <none>
    Args:
      --csi-address=$(ADDRESS)
      --v=5
      --enable-leader-election
      --leader-election-type=leases
    State:          Running
      Started:      Wed, 24 Feb 2021 14:30:37 +0530
 ```